metric/blockgroup/rCOB.csv

```diff
@@ -1,154 +1,154 @@
+370630017053,2,5,2,5,3,2,0,2,2
+370630017061,0,0,1,0,1,1,0,0,0
+370630017062,1,0,0,0,1,0,0,1,0
+370630017071,0,0,0,0,0,0,0,0,0
+370630017072,5,4,1,2,3,3,2,4,2
+370630017073,0,0,0,0,0,0,0,0,0
+370630017074,0,0,0,0,0,0,1,0,1
+370630017075,0,0,0,1,1,1,4,2,2
+370630017081,1,0,0,3,3,1,0,1,9
+370630017091,0,0,0,0,0,0,0,0,0
+370630017092,0,0,0,1,0,0,0,0,0
+370630017093,5,5,9,6,5,6,8,4,1
+370630017101,0,2,3,8,4,3,3,2,1
+370630017102,0,0,0,0,0,0,0,0,0
+370630017111,0,0,0,1,3,1,0,1,1
+370630017112,0,0,0,0,0,0,0,0,2
+370630017113,0,0,0,0,1,0,0,0,0
+370630018011,1,0,0,1,0,0,3,0,0
+370630018012,0,0,0,0,0,1,0,1,0
+370630018013,0,1,0,0,2,1,0,0,0
+370630018014,2,1,4,4,4,4,1,3,2
+370630018015,0,0,3,4,1,0,5,1,6
+370630018021,1,1,0,1,0,1,2,1,0
+370630018022,0,0,0,0,0,0,1,0,0
+370630018023,1,1,0,1,1,0,1,0,0
+370630018024,0,0,0,0,1,1,0,0,0
+370630018061,0,1,0,1,0,0,0,0,0
+370630018062,0,0,0,0,1,0,0,1,0
+370630018071,1,0,3,0,0,0,0,0,0
+370630018072,0,0,0,0,0,0,0,0,0
+370630018073,0,0,0,0,0,0,0,1,0
+370630018081,0,0,0,1,1,0,0,0,0
+370630018082,2,0,1,5,2,2,0,1,0
+370630018091,10,14,7,8,6,3,4,2,3
+370630018092,0,0,1,2,4,3,0,2,0
+370630019001,0,1,1,1,2,2,3,2,1
+370630019002,0,0,0,0,1,0,0,0,0
+370630020071,0,0,0,0,0,0,0,0,1
+370630020072,4,1,3,4,2,2,2,2,0
+370630020073,0,0,0,0,0,0,0,0,0
+370630020081,0,0,0,0,0,0,0,0,0
+370630020082,0,0,1,0,1,3,2,1,0
+370630020091,1,2,0,5,2,2,2,3,2
+370630020092,0,0,1,0,1,0,0,0,1
+370630020093,1,0,0,2,3,1,1,0,0
+370630020131,0,1,1,0,1,0,0,0,0
+370630020132,0,1,0,0,0,0,0,0,0
+370630020133,0,0,0,0,0,0,0,0,0
+370630020151,1,2,8,5,0,1,2,0,0
+370630020152,4,5,7,11,6,2,2,15,8
+370630020153,14,9,9,10,5,4,2,4,10
+370630020161,2,3,0,2,0,1,0,1,1
+370630020162,15,11,13,9,21,12,12,10,8
+370630020171,0,1,0,0,0,2,1,0,0
+370630020172,0,1,0,0,0,2,21,1,1
+370630020173,5,7,4,5,5,2,4,3,6
+370630020181,0,1,0,1,0,0,0,0,0
+370630020182,5,5,5,8,7,4,6,8,2
+370630020183,10,9,7,6,3,1,0,3,4
+370630020191,10,13,16,23,19,4,6,7,4
+370630020192,0,0,1,1,0,0,0,0,0
+370630020193,2,0,0,0,1,0,1,1,0
+370630020201,3,1,2,1,1,2,1,2,1
+370630020202,0,0,1,0,1,1,0,0,0
+370630020211,66,29,15,40,19,29,39,21,19
+370630020212,1,2,0,1,0,1,1,0,0
+370630020221,1,0,2,1,4,6,13,1,1
+370630020222,0,0,1,0,0,0,0,0,0
+370630020231,1,0,0,0,0,0,0,0,0
+370630020232,7,3,5,3,3,3,7,2,6
+370630020241,1,0,0,0,0,0,0,0,2
+370630020242,0,0,0,0,0,0,0,0,0
+370630020243,0,0,0,0,0,0,0,0,0
+370630020251,1,0,0,0,0,0,1,0,0
+370630020252,0,0,0,1,0,1,2,0,2
+370630020253,0,0,0,0,0,0,0,0,0
+370630020261,0,0,1,0,0,1,0,0,0
+370630020262,1,0,1,1,4,1,0,4,0
+370630020271,7,8,6,5,15,4,8,10,8
+370630020272,25,35,24,35,24,28,34,22,20
+370630020273,0,0,0,0,1,1,1,0,1
+370630020281,7,11,16,22,14,29,23,18,19
+370630020282,0,1,0,1,0,0,2,0,0
+370630020283,29,44,51,77,76,46,65,51,52
+370630021001,14,11,6,12,3,3,4,7,6
+370630021002,0,0,0,0,0,1,0,0,0
+370630021003,0,0,0,0,1,0,0,0,1
+370630022001,33,41,53,46,41,44,40,49,39
+370630023001,6,7,15,10,6,6,6,8,6
+370630023002,1,1,1,3,1,0,1,0,1
+370639801001,63,76,83,50,53,36,69,65,46
```

Comment:
One 2019 value changed?

Reply:
I'm not sure why this went from 62 to 65. Looking at the raw permit data that is in PostGIS now, I see 65 permits in 2019 in that blockgroup, so that value is correct. The only change i made which might have impacted it is doing some data cleaning to fill in occupancy descriptions where they were missing (based on the occupancy codes), so it's possible that a few of the 2019 permits were missing values there and thus weren't counted?